package.json

```diff
@@ -24,9 +25,11 @@
         "@types/mongodb": "^3.6.16",
         "@uiw/react-md-editor": "^3.4.11",
         "add": "^2.0.6",
+        "aws-amplify": "^4.1.2",
         "axios": "^0.21.1",
         "faker": "^5.5.3",
         "framer-motion": "^3.2.1",
+        "global": "^4.4.0",
```

Comment:
why was this package added?

Reply:
I think it was accidently added for something we were trying to do earlier